flepimop/gempyor_pkg/src/gempyor/testing.py

```diff
@@ -5,10 +5,21 @@
 the optional test dependencies must be installed.
 """
 
+__all__ = [
```

Comment:
seems to just be exporting everything; is that not the default?

or are you just being deliberately explicit here?

Reply:
This is not exporting everything, it's exporting a very explicit subset of the module contents. I think by default anything that does not begin with an underscore is exported. This excludes things like `confuse`, `np`, `pytest`, etc. from being exported. A quick example (contrasting to `gempyor.seir` which does not have an `__all__`):
```python
>>> from gempyor.testing import *
>>> np
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
NameError: name 'np' is not defined
>>> from gempyor.seir import *
>>> np
<module 'numpy' from '/Users/twillard/Desktop/GitHub/HopkinsIDD/flepiMoP/venv/lib/python3.11/site-packages/numpy/__init__.py'>
```

Reply:
ah, so the __all__ precludes anything that this module itself has imported from re-export. got it.

i'm a bit surprised that there isn't a e.g. `@export` decorator, but cursory search indicates not.

Reply:
Yeah, unfortunately export management is an area where python underwhelms.